build/version.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
 
     <!-- package version of grpc-dotnet -->
-    <GrpcDotnetVersion>2.31.0-dev</GrpcDotnetVersion>
+    <GrpcDotnetVersion>2.32.0-pre1</GrpcDotnetVersion>
```

Comment:
GrpcDotnetAssemblyFileVersion in this file should be 2.32.0.0 as well

Reply:
thanks, done